db/analytics_migrate/20210915160357_create_ecf_school.rb

```diff
@@ -2,7 +2,7 @@
 
 class CreateECFSchool < ActiveRecord::Migration[6.1]
   def change
-    create_table :ecf_schools do |t|
+    create_table :ecf_schools do |t| # rubocop:disable Rails/CreateTableWithTimestamps
```

Comment:
Do we need to add `# rubocop:enable Rails/CreateTableWithTimestamps` in the end of this migration file?

Reply:
if `# rubocop:disable` is added to a new line without code yes, however if its added to a block or line of code it will only apply to that block/line and not need to be re-enabled. See https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code 

I tested also locally and if you need it there is a cop that will fail as well: 
```
Lint/MissingCopEnableDirective: Re-enable Rails/CreateTableWithTimestamps cop with # rubocop:enable after disabling it.
```